.circleci/config.yml

```
@@ -168,9 +212,19 @@ workflows:
             tags:
               only: /.*/
 
-      - deploy_package:
+      - deploy_pypi:
+          requires:
+            - test_package
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /.*/
+
+      - deploy_docker:
           requires:
             - test_package
+            - deploy_pypi
```

You want to make sure your image is built.

```suggestion
            - build
            - deploy_pypi
```